perf/common.jl

```diff
@@ -0,0 +1,32 @@
+"""
+    AtmosCoveragePerfConfig()
+    AtmosCoveragePerfConfig(; s, config_dict)
+
+Creates a model configuration that covers many physics components.
+
+The configuration precedence is as follows:
+1. Configuration from the given config file/dict (highest precedence)
+2. Default perf configuration (to increase coverage)
+4. Default configuration (lowest precedence)
+"""
```

Comment:
This should be updated: `s` is no longer an accepted argument, and `4.` should become `3.`